### ISSUE TITLE
Add HTTPS

### ISSUE THREAD

**Issue by konklone** [meiqimichelle/mhertzfeld#17]
As discussed, give CloudFlare's New and Free :boom: [Universal SSL](https://blog.cloudflare.com/introducing-universal-ssl/) a shot on your blog.

It's [not a perfect solution](https://konklone.com/post/github-pages-now-supports-https-so-use-it), but it's way better than nothing and it gets you into the encrypted web.

**Comment by meiqimichelle** [comment]
Updated site's nameservers to CloudFlare; waiting for certificate provision to make changes to site internal links, etc.

**Comment by konklone** [comment]
https://michellehertzfeld.com/ works! But is unstyled. I fixed this in #18. After that, all you need to do is tell Cloudflare to force a redirect from `http://` to `https://` for the main site.

**Comment by meiqimichelle** [comment]
Ok, after #25, we're in business at https://michellehertzfeld.com and its various permutations! Woot. Now, on to Cloudflare...

**Comment by meiqimichelle** [comment]
Wooooooo :confetti_ball: I think this is done. By which I mean.... http://mhertzfeld.com and http://michellehertzfeld.com both redirect to https://michellehertzfeld.com, and I gots styles and everything. My understanding is that I can't control http://meiqimichelle.github.io/mhertzfeld/ easily (ie, I can't automatically redirect it anywhere without a little JS snippet or other hacky thing). I'm opening a separate ticket for that: #26.

**Comment by meiqimichelle** [comment]
I ended up implementing the redirect in #26. I think we're all done here, nothing more to see, go about your business :-/ I'm gonna go ahead and close because, well, @konklone you've got a lot more going on than my little blog! Re-open if you see something amiss.

**Comment by konklone** [comment]
Nice! :+1: :tada:
